app/templates/partials/base.html

```diff
@@ -17,7 +17,8 @@
   <link href="{{ url_for('static', path='/global.css') }}" rel="stylesheet">
 
   <!-- Icons -->
-  <script src="https://unpkg.com/ionicons@5.2.3/dist/ionicons.js"></script>
+  <script type="module" src="https://unpkg.com/ionicons@5.4.0/dist/ionicons/ionicons.esm.js"></script>
+  <script nomodule="" src="https://unpkg.com/ionicons@5.4.0/dist/ionicons/ionicons.js"></script>
```

Comment:
Move all the JS and audio to be before the closing of the body

Reply:
Done